test/DGmethods/Euler/euler_model.jl

```diff
@@ -0,0 +1,146 @@
+using CLIMA.VariableTemplates
+using StaticArrays
+
+import CLIMA.DGmethods: BalanceLaw, vars_aux, vars_state, vars_gradient,
+                        vars_diffusive, flux!, source!, wavespeed,
+                        boundarycondition!, gradvariables!, diffusive!,
+                        init_aux!, init_state!, init_ode_param, init_ode_state
+
+abstract type EulerProblem end
+
+struct EulerModel{P, G, R} <: BalanceLaw
+  problem::P
+  gravity::G
+  refstate::R
+  function EulerModel(problem;
+                      gravity,
+                      refstate=NoReferenceState())
+    P = typeof(problem)
+    G = typeof(gravity)
+    R = typeof(refstate)
+    new{P, G, R}(problem,gravity, refstate)
+  end
+end
+
+init_state!(m::EulerModel, x...) = initial_condition!(m, m.problem, x...)
+
+function vars_aux(m::EulerModel, T)
+  NamedTuple{(:refstate, :gravity),
+             Tuple{vars_aux(m.refstate, T), vars_aux(m.gravity, T)}}
+end
+
+function vars_state(::EulerModel, T)
+  NamedTuple{(:ρ, :ρu⃗, :ρe), Tuple{T, SVector{3, T}, T}}
+end
+
+vars_gradient(::EulerModel, T) = Tuple{}
+vars_diffusive(::EulerModel, T) = Tuple{}
+
+const γ_exact = 7 // 5
+
+function pressure(ρ::T, ρinv, ρe, ρu⃗, ϕ) where T
```

Comment:
Just a reminder, `ρu⃗` is not rendering. So it's a bit difficult to review.

Reply:
Do you have a better suggestion? In my mind, using just `ρu` is a bit confusing since this is a vector quantity.

Reply:
Yeah, @simonbyrne has just been using `ρu` anyway. IMO, `\vec` is a portability issue since it doesn't 'port' to the browser. @asraero pointed out that bold works :/ `ρ𝐮`. I'm torn, I wish the vector _did_ render because I think it's the clearest in the editor, so I don't have strong feelings either way.

I just figured it might be a good thing to review. Let's stick with it for now, I'll raise an issue to see if it's worth changing.